tools/isobuild/bundler.js

```diff
@@ -1522,11 +1522,22 @@ class ClientTarget extends Target {
     if (this.arch === 'web.cordova') {
       const { WebAppHashing } =
         loadIsopacket('cordova-support')['webapp-hashing'];
+
       const cordovaCompatibilityVersions =
         _.object(_.map(CORDOVA_PLATFORM_VERSIONS, (version, platform) => {
-          const hash = WebAppHashing.calculateCordovaCompatibilityHash(
-            version,
-            this.cordovaDependencies);
+
+          const pluginsExcludedFromCompatibilityHash = (process.env.METEOR_CORDOVA_COMPAT_VERSION_EXCLUDE || '')
+            .split(',');
```

Comment:
Yeah, I know. It's not producing any side effects in `_.omit` so I'm okay with this. Otherwise, it would look like `process.env.METEOR_CORDOVA_COMPAT_VERSION_EXCLUDE ? process.env.METEOR_CORDOVA_COMPAT_VERSION_EXCLUDE.split(',') : []`. Or with an extra line to assign this env var to sth shorter 🙂